src/ResourceManager/LogicApp/Commands.LogicApp/Cmdlets/IntegrationAccount/UpdateAzureIntegrationAccountGeneratedIcnCommand.cs

```diff
@@ -0,0 +1,95 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+namespace Microsoft.Azure.Commands.LogicApp.Cmdlets
+{
+    using System;
+    using System.Globalization;
+    using System.Management.Automation;
+    using Microsoft.Azure.Commands.LogicApp.Utilities;
+
+    /// <summary>
+    /// Updates the integration account generated interchange control number.
+    /// </summary>
+    [Cmdlet(VerbsCommon.Set, "AzureRmIntegrationAccountGeneratedIcn", SupportsShouldProcess = true)]
+    [OutputType(typeof(IntegrationAccountClient.IntegrationAccountControlNumber))]
```

Comment:
In other words - you need to provide a simple type definition, not a type defined within another type.